environment-ros3.yml

```diff
@@ -6,10 +6,10 @@ channels:
 dependencies:
   - python==3.12
   - h5py==3.10.0
```

Comment:
```suggestion
  - h5py==3.11.0
```

Reply:
Was this was the package version you mentioned wasn't on conda yet? I think it might be now so I updated to match `requirements.txt`

Reply:
I don't believe I said that, but looking at the feedstock, it is now on conda.